app/views/applications/index.html

```diff
@@ -18,16 +18,20 @@ <h1 class="govuk-heading-l">{{ title }}</h1>
         <p class="govuk-body">You can add up to 4 applications at a time.</p>
 
         {% if not (data | allSectionsCompleted) %}
-          <p class="govuk-body">You will not be able to submit applications until you have completed your details.</p>
+          <p class="govuk-body">You will not be able to submit applications until you have completed <a href="/details">your details</a>.</p>
         {% endif %}
       {% else %}
         <p class="govuk-body">You can add {{ numberOfApplicationsLeft }} more {{ "applications" if  numberOfApplicationsLeft > 1 else "application" }}.</p>
       {% endif %}
 
+      {% if (applicationsSubmitted | length == 0) %}
+        <p class="govuk-body">You’ll be asked to write a personal statement of 500 words before you submit.</p>
```

Comment:
You'll be asked to write a personal statement of 500 words before you can submit your application.